ui/pages/notifications/notifications.tsx

```diff
@@ -54,30 +49,9 @@ export const enum TAB_KEYS {
   WEB3 = 'notifications-other-tab',
 }
 
-// Cleanup method to ensure we aren't keeping really old notifications.
-// See internals to tweak expiry date
-const useEffectDeleteExpiredNotifications = () => {
```

Comment:
Maybe this was meant to solve this problem? https://github.com/MetaMask/metamask-extension/pull/27975/files#r1834386407